scripts/src/public-api-methods.ts

```diff
@@ -0,0 +1,249 @@
+// https://api.slack.com/methods
+// These are all of the public Slack API methods
+export const getPublicAPIMethods = () => {
+  const methods = [
```

Comment:
nit: I'm not sure how this array was populated, but I was wondering if we should make it a `Set` to ensure uniqueness (in case there is a dupe accidentally added)